gwswlib/threedi.py

```diff
@@ -44,10 +44,44 @@ def __init__(self):
     def import_hydx(self, hydx):
         self.connection_nodes = []
         self.manholes = []
+        self.connections = []
+        self.structures = []
 
         for connection_node in hydx.connection_nodes:
+            check_if_element_is_created_twice(
```

Comment:
In hoeverre is dit eigenlijk `check_if_element_already_exists()`?
Qua naam klinkt het als "we gaan nu een setje nodes creeëren, controleer bij die nieuwe nodes of er daar een dubbele tussen zit".